wordpress-mobile/WordPress-Android
# UI Modernization Post: remove post list default actions

Parent #19341 

This PR removes the logic surrounding "default" actions as all buttons have been removed from the post list item. `SingleItem` relates to each action item within the more menu.

There are no UI changes in this PR.

To test:
- Install the app
- Login and select a site
- Navigate to My Site > Posts
- Tap on the tabs and context menus
- ✅ Verify all still works as expected

## Regression Notes
1. Potential unintended areas of impact
Post list context menu items are not shown as expected

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Updated PostListItemUiStateHelperTest

3. What automated tests I added (or what prevented me from doing so) N/A

PR submission checklist:

UI Changes testing checklist: N/A
